deposit/src/main/java/edu/unc/lib/deposit/validate/ExtractTechnicalMetadataJob.java

```diff
@@ -274,8 +276,8 @@ private Element getFitsIdentificationInformation(Document fitsDoc) {
             // Trust any answer agreed on by multiple tools
             for (Element el : identification.getChildren("identity", FITS_NS)) {
                 if (el.getChildren("tool", FITS_NS).size() > 1
-                        || (!"Exiftool".equals(el.getChild("tool", FITS_NS).getAttributeValue("toolname"))
-                                && !"application/x-symlink".equals(el.getAttributeValue("mimetype")))) {
+                        || !("Exiftool".equals(el.getChild("tool", FITS_NS).getAttributeValue("toolname"))
```

Comment:
It looks like we may have a logic problem here, since ~p & ~q is not equivalent to ~(p & q). Maybe the logic was wrong to begin with, but just thought I'd call this out :-)